cartographer_rviz/cartographer_rviz/drawable_submap.cc

```diff
@@ -159,13 +168,16 @@ void DrawableSubmap::SetAlpha(const double current_tracking_z) {
 
 void DrawableSubmap::UpdateSceneNode() {
   ::cartographer::common::MutexLocker locker(&mutex_);
+  render_in_progress_ = false;
+  if (quitting_) {
```

Comment:
Have a look at my reply to @damonkohler: https://github.com/googlecartographer/cartographer_ros/pull/289#discussion_r108674387